src/s3fs.cpp

```diff
@@ -59,6 +59,31 @@
 #define ENOATTR                   ENODATA
 #endif
 
+#define PRINT_FUSE_CTX(level, indent, fmt, ...) do {                \
```

Comment:
Could this macro be written in `s3fs_logger.h`?
I think it's easier to understand.

Reply:
I thought about it too, but I decided to put the macros in s3fs.cpp instead of s3fs_logger.h, as these macros are only used in s3fs.cpp. But I don't have strong preference, I can move them to s3fs_logger.h if you confirm.

Reply:
I can understand your thoughts and judgments. (me too)
I think that since this is a macro for outputting messages according to the debug log level, it seems easier to understand if they are grouped in s3fs_logger.h.
It would be helpful if you could move the macro description.